### ISSUE TITLE
Allow multiple device IDs in `impt device assign`

### ISSUE THREAD

**Issue by deldrid1** [EatonGMBD/imp-central-impt#7]
I'd like to allow multiple device IDs to be passed into `impt device assign`, which I think is supported by the underlying API?

Instead of doing a bunch of serial requests 
```shell
impt device assign -d pN6z2LE9H8u4 -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 82Auv9f6Xtf6 -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d WRA6wpc3dFGY -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d -ZDervOZyQbL -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d vjgq9-SdGOw7 -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d Qpv-MM9Hr2-K -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 3-vV7lKqDfqt -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d ppp_rDucAYtd -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d Nrfa0CPKvFnU -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d LRyRxSzqVM0T -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d hdneZ93A6eYY -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d CW2JWDDRgc3F -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 75bKElRgZkKn -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d bp5HH4o7hjnE -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d B5lwXAwoCS4F -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d TINiimSWsPwc -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d KshsCFf4kK9L -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 9tKqQR8Tkglb -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 9xuoyF5GelLV -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d KD6ShRhg6MjR -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d lD1giC9vaXIo -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 6esv4fMP7ibB -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 135b_-Qi-ax- -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d Xg2_l2aTX8-G -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d aOG1ui5RVova -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d uBdLElfR5C31 -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d xvCy-_AL4AwG -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 3Bv-Vk6RWDs- -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d 2kB8_PsdPaxh -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d CDmjEVloMebo -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d Nbf06urvQDwq -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;impt device assign -d ZtEnzpeo1HTK -g 7c4b5300-282e-0eba-3d08-854251674a23 -q;
```

You could do one large parallel request:

```shell
impt device assign -g 7c4b5300-282e-0eba-3d08-854251674a23 -q -d  30000c2a690e1c66 30000c2a690e1cf6 30000c2a690e1ada 30000c2a690e19a0 30000c2a690e2c1f 30000c2a690e1a07
```

**Comment by alexey-nbl** [comment]
@deldrid1 `-d  30000c2a690e1c66 30000c2a690e1cf6 30000c2a690e1ada` will be hard/impossible to support (yargs limitation) and it will not be aligned with syntax of other impt commands.
But   `-d  30000c2a690e1c66 -d 30000c2a690e1cf6 -d 30000c2a690e1ada` should be possible (and similar idea is already used in some other impt commands).

**Comment by deldrid1** [comment]
Works for me!

**Comment by mm-gmbd** [comment]
Fixed with #15
